pylint/checkers/base/comparison_checker.py

```diff
@@ -58,6 +58,11 @@ class ComparisonChecker(_BasicChecker):
             "comparison-with-itself",
             "Used when something is compared against itself.",
         ),
+        "R0125": (
+            "Redundant comparison - %s",
+            "comparison-of-two-literals",
+            "Used when two literals are compared with each other",
```

Comment:
```suggestion
            "When two literals are compared with each other the result is a constant, "
            "and using the constant directly is both easier to read and more performant."
            " Initializing 'True' and 'False' this way is not required since python 2.3",
```